app/src/client/components/buildPage/UserProperty.tsx

```diff
@@ -54,8 +70,8 @@ export const UserProperty = memo(({ activeProperty, propertiesSchema, sideNavIte
   };
 
   useEffect(() => {
-    setActiveModel(null);
-  }, [sideNavItemClickCount]);
+    clearStack();
+  }, [sideNavItemClickCount, clearStack]);
```

Comment:
why dependency on clearStack function?

Reply:
Yup, we don't need the clearStack function as part of the dependency.